zenmap/share/zenmap/locale/es.po

```diff
@@ -0,0 +1,2597 @@
+"valor clickeando en el campo de valores al lado del nombre del argumento."
+
+#: zenmapGUI/ScriptInterface.py:483
+msgid "Please wait."
+msgstr "Espere por favor."
+
+#: zenmapGUI/ScriptInterface.py:507
+msgid "Names"
+msgstr "Nombres"
+
+#: zenmapGUI/ScriptInterface.py:628
+msgid "Arguments"
+msgstr "Argumentos"
+
+#: zenmapGUI/ScriptInterface.py:721
+msgid "Select script files"
+msgstr "Seleccionar archivos de scripts"
+
+#: zenmapGUI/FilterBar.py:24
+msgid "Host Filter:"
+msgstr "Filtro de Host:"
+
+#: zenmapGUI/FilterBar.py:57
+msgid ""
+"Entering the text into the search performs a <b>keyword search</b> - the "
+"search string is matched against every aspect of the host.\n"
+"\n"
+"To refine the search, you can use <b>operators</b> to search only specific "
+"fields within a host. Most operators have a short form, listed. \n"
+"<b>target: (t:)</b> - User-supplied target, or a rDNS result.\n"
+"<b>os:</b> - All OS-related fields.\n"
+"<b>open: (op:)</b> - Open ports discovered in a scan.\n"
+"<b>closed: (cp:)</b> - Closed ports discovered in a scan.\n"
+"<b>filtered: (fp:)</b> - Filtered ports discovered in scan.\n"
+"<b>unfiltered: (ufp:)</b> - Unfiltered ports found in a scan (using, for "
+"example, an ACK scan).\n"
+"<b>open|filtered: (ofp:)</b> - Ports in the \"open|filtered\" state.\n"
+"<b>closed|filtered: (cfp:)</b> - Ports in the \"closed|filtered\" state.\n"
+"<b>service: (s:)</b> - All service-related fields.\n"
+"<b>inroute: (ir:)</b> - Matches a router in the scan's traceroute output.\n"
+msgstr ""
+"Introduciendo el texto en el campo de búsqueda ejecuta una <b>búsqueda de palabra clave"
+"</b> - la búsqueda de la cadena de caracteres es comprobada contra cada aspecto "
+"del servidor.\n"
+"\n"
+"Para mejorar la búsqueda, puede usar <b>operadores</b> para buscar únicamente "
+"campos específicos dentro del servidor. Gran parte de los operadores tienen "
+"una forma corta, listada a continuación. \n"
+"<b>target: (t:)</b> - Datos proporcionados por el usuario, o un resultado rDNS.\n"
+"<b>os:</b> - Todos los campos relacionados con el SO.\n"
+"<b>open: (op:)</b> - Puertos abiertos encontrados en un escaneo.\n"
+"<b>closed: (cp:)</b> - Puertos cerrados encontrados en un escaneo.\n" 
+"<b>filtered: (fp:)</b> - Puertos filtrados encontrados en un escaneo.\n"
+"<b>unfiltered: (ufp:)</b> - Puertos sin filtrar encontrados en un escaneo "
+"(utilizando por ejemplo un escaneo ACK).\n"
+"<b>open|filtered: (ofp:)</b> - Puertos en estado \"abierto|filtrado\".\n"
+"<b>closed|filtered: (cfp:)</b> - Puertos en estado \"cerrado|filtrado\".\n"
+"<b>service: (s:)</b> - Todos los campos relacionados a los servicios.\n"
+"<b>inroute: (ir:)</b> - Busca un router en la salida del traceroute del escaneo.\n"
+
+#: zenmapGUI/ScanHostsView.py:168
+msgid "Hosts"
+msgstr "Servidores"
+
+#: zenmapGUI/ScanHostsView.py:169 radialnet/gui/NodeNotebook.py:196
+msgid "Services"
+msgstr "Servicios"
+
+#: zenmapGUI/ScanHostsView.py:180
+msgid "OS"
+msgstr "SO"
```

Comment:
OS is pretty obvious for everyone nowadays, thus (since I corrected the French version) I would leave "OS (SO)" or simply "OS" like I did in French. Do you agree with that?